sdk/nodejs/src/eventQueue.ts

```diff
@@ -36,14 +36,14 @@ export type EventQueueOptions = {
     flushEventQueueSize?: number,
     logger: DVCLogger,
     reporter?: DVCReporter,
-    baseURLOverride?: string
+    eventsAPIBaseURLOverride?: string
```

Comment:
how is this making it from client initialization into here?

Reply:
and shouldn't this just use the same name as the outer client option?

Reply:
Yeah - missed this in naming updates

Reply:
it's honestly kind of weird that this class defines its own interface for the options when all of these options are supposed to come directly from the DVCOptions object..

Reply:
🤷🏻